1-WebApp-OIDC/1-5-B2C/README.md

```diff
@@ -100,6 +108,9 @@ If your web site needs to be accessed from users using iOS 12, you probably want
 
 ## About The code
 
+#### Where is MSAL?
+This sample does NOT use MSAL - it uses the built-in ASP.NET Core middlerware. MSAL is used for fetching access  for accessing protected APIs (not shown here), as well as ID tokens. For logging-in purposes, it is sufficient to obtain an ID Token, and the middlerware is capable of doing this on its own.
```

Comment:
This sample does NOT use MSAL as it only signs-in users (it does not call a Web API).